index.html

```diff
@@ -7,15 +7,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <!-- CSS STYLE SHEET -->
     <link rel="stylesheet" href="style.css">
-    <title>My Portfolio</title>
+    <title>Portfolio : Ghaus Ali Khan</title>
```

Comment:
- [ ] Please use the correct `title` which looks more professional and helps any other person to see the user name even with less tab width and don't use any other characters except `|` or `-`. Kindly look at the below suggestion for better understanding.

```suggestion
    <title>Ghaus Khan | Portfolio</title>
```